colossalai/engine/ophooks/_memtracer_ophook.py

```diff
@@ -25,6 +23,7 @@ class MemTracerOpHook(BaseOpHook):
     """
 
     def __init__(self, warmup: int = 50, refreshrate: int = 10, data_prefix: str = "memstats"):
+        from colossalai.gemini.memory_tracer import AsyncMemoryMonitor
```

Comment:
Why import in function?

Reply:
circle import. hard to fix.

Reply:
circle import.
We will fix the import in the next PR